docs/pages/getting-started.html

```diff
@@ -9,7 +9,7 @@
 
 <h3 class="usa-heading">For developers</h3>
 
-<p>The UI components are built on a solid <strong>HTML</strong> foundation, progressively enhanced to provide core experiences across browsers. All users get critical information and experiences. New browsers get the prettiest experiences, while older browsers get less pretty, but usable ones. If JavaScript fails, users will still get a robust HTML foundation.</p>
+<p>The UI Components are built on a solid <strong>HTML</strong> foundation, progressively enhanced to provide core experiences across browsers. All users get critical information and experiences. New browsers get the prettiest experiences, while older browsers get less pretty, but usable ones. If JavaScript fails, users will still get a robust HTML foundation.</p>
```

Comment:
@maya Howdy! My thought was that "UI Components" (as we're using it) is a proper noun — it refers to *our* components. 

If the components being referenced here are a more general, non-WDS set of components, lowercasing is the way to go.